.github/workflows/unlabel-review.yml

```diff
@@ -0,0 +1,22 @@
+
+name: Reviewed
+# Runs when a review is submitted to a PR and
+# remove the "Waiting for Reviewer" label
+on:
+  pull_request_review:
```

Comment:
I believe `pull_request_target` is the only event that has access to `secrets.GITHUB_TOKEN` and we need this to delete the label.

Currently, I do think think any of the `pull_request_target` [activity types](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target) triggers with a new review.

An alternative is to use a cron based github action, which would complicate things.

Reply:
To be explicit: admin rights are required to retrieve the list of the members of the core-dev team in the scikit-learn organization.

Reply:
It works on your own repo because pull request in your repo has access to `secrets.GITHUB_TOKEN`. This means it will not have access to the teams api or have access to deleting a label.

A pull request from a fork will not have access to `secrets.GITHUB_TOKEN`. You can read more about [this topic here](https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/)

Reply:
`pull_request_target` runs in the context of the master branch, which has code already approve and thus can have admin rights.

In general, giving `pull_request_target` admin rights would be dangerous because it will give admin rights to any person that opens a PR.

Reply:
> It works on your own repo because pull request in your repo has access to `secrets.GITHUB_TOKEN`.
> 
> A pull request from a fork will not have access to `secrets.GITHUB_TOKEN`. This means it will not have access to the teams api or have access to deleting a label.
> 
> You can read more about [this topic here](https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/)

Thanks @thomasjpfan I understand that... It will work on scikit-learn once merged. This is clear to me. That's why I tested it on my repo and I'm not expecting it works on scikit-learn.
Now, I can't test the selection of core-devs on my repo unless I create an organization and recreate the same structure as the scikit-learn one. I can't test my workflow locally neither because I can't  retrieve the members of the core-dev  team. I'm happy to have any suggestion about that.
Is my issue clear enough?

Reply:
> `pull_request_target` runs in the context of the master branch, which has code already approve and thus can have admin rights.
> 
> In general, giving `pull_request_review` admin rights would be dangerous because it will give admin rights to any person that opens a PR.

I'm not asking for admin rights I'm asking for suggestions to test this implementation.

Reply:
> I'm not asking for admin rights I'm asking for suggestions to test this implementation.

Sorry, I was not clear. I do not think this PR will not work even if it is merged with scikit-learn.

Reply:
> I'm not asking for admin rights I'm asking for suggestions to test this implementation.

I am thinking of admin rights to the github action and not a specific user.

Reply:
> I'm asking for suggestions to test this implementation.

I would create a new public organization just for testing with yourself on a team and add this implementation there. Then create a public repo in that organization and make a fork of that repo. Lastly, create a PR from your fork and check to see if the github action works.

You are open to try this out. I suspect that this will not work because of how github actions does permissions.

Reply:
>  would create a new public organization just for testing with yourself on a team. Then create a public repo in the organization, add this implementation to that repo and make a fork. Lastly, create a PR from your fork and check to see if the github action works.
> 
> You are open to try this out. I suspect that this will not work because of how github actions does permissions. (I have ran into this type of issue a few times when working with github actions)

Ok, thanks @thomasjpfan! I will do it and ping you and @alfaro96 (if he's ok with that) to play around.. :)